docs/source/hardware/lowpriority.rst

```diff
@@ -0,0 +1,75 @@
+
+.. _partition_names:
+
+**********
+Low priority partition
+**********
+:sub:`Updated September, 2022`
+
+
+Introduction
+===================
+``lowpriority`` is a new partition on our high performance computing (HPC) cluster - Discovery, that allows the research community to use private resources on the HPC cluster when they are idle. 
```

Comment:
I think we should remove the reference to Discovery, just to save us from having to update when we switch to Explorer. So, it could just read, ". . . on our high performance computing (HPC) cluster that allows the research community. . ."

If you want to keep the reference to Discovery, I suggest taking out the hyphen between cluster and Discovery, replacing it with a comma, and changing 'that' to 'which'.

Reply:
Thanks @tketchem14 , we did have a conversation about this earlier but I think a lot of our users will actually be more familiar with/relate to "Discovery" than with "high performance computing" so it might be okay to leave it there. We can certainly replace this when we are closer to bringing up the new cluster. Let me know if this doesn't sound okay to you and we can discuss further.

Reply:
@ShobanaSekar , can you pull out the hyphen between cluster and Discovery to address @tketchem14's comment and mark this as resolved? Thanks.